atlp-rwanda/atlp-pulse-fn
# #DP-108(Feat):Fix URI Naming

### PR description 
_Fixing app URI naming_
### What this PR does
- Fix login URI name
- Fix some dashboard redirects URI names
### How can it be tested
Go to the app, and navigate to the login. The uri should now be `/login`. After login, you will navigate to the dashboard and every link should be on its own without having the `/dashboard` in front.
### Background context
Successfully merging this PR will introduce new names in the links that are more realistic and easy to understand by any user
### Relevant trello ID
[#DP-108](https://trello.com/c/EQl3uwYN/108-fix-login-uri)
### Screenshot
_N/A_
### Checks completed
# Please check this Checklist before you submit your PR:
